nomenclature/testing.py

```diff
@@ -24,3 +34,15 @@ def assert_valid_yaml(path):
         raise AssertionError(
             "Parsing the yaml files failed. Please check the log for details."
         )
+
+
+def assert_valid_structure(path):
+    """Assert that "definition" folder in `path` can be initialized without errors"""
+
+    # check if the directory has a sub-folder "definitions"
+    for name in Path(path).glob("**"):
```

Comment:
Hi Philip, thanks for your review! 
Ah yes, I don't need to iterate over the whole path, simply checking is cleaner, thanks!
Having a warning "No folder `definitions` found in the directory", and if the folder exists having an assertion "The folder `definitions` can't be parsed into a DataStructureDefinition", as partially discussed with @danielhuppmann in Teams, might be smoother than directly throw an assertion if there is no folder. But on the other hand I also don't know if one needs to have a folder `definitions`, then two assertions make sense.

Reply:
Yes, I think that having two separate assertions is cleaner.

As a bonus (because I ran into this error yesterday with the EMF37 project): write a logger-warning if the parsed `DataStructureDefinition` has empty `variable` or `region` attributes (maybe as part of the DataStructureDefinition-initialization).